### ISSUE TITLE
standalone version error 

### ISSUE THREAD

**Issue by SusiStinkemuehle** [hMRI-group/hMRI-toolbox#89]
Hi, 
i want to start the new released standalone version of your hMRI toolbox under Ubuntu 22.04, i installed the matlab runtime 2023a and unpacked the hMRI standalone toolbox. 
When i try to start the hMRI toolbox, the following error occurs:

sudo ./standalone-hMRItoolboxv0.6.1/run_spm12.sh ./MCR/R2023a/
------------------------------------------
Setting up environment variables
---
LD_LIBRARY_PATH is .:./MCR/R2023a//runtime/glnxa64:./MCR/R2023a//bin/glnxa64:./MCR/R2023a//sys/os/glnxa64:./MCR/R2023a//sys/opengl/lib/glnxa64
Unrecognized function or variable 'matlabrc'.

 ___  ____  __  __
/ __)(  _ \(  \/  )
\__ \ )___/ )    (   Statistical Parametric Mapping
(___/(__)  (_/\/\_)  SPM12 - https://www.fil.ion.ucl.ac.uk/spm/

SPM12: v7771                                       12:04:15 - 10/12/2023
========================================================================
Initialising SPM                        :                   ..Error using spm_select
Undefined function 'cfg_getfile' for input arguments of type 'cell'.

Error in spm_select (line 110)

Error in spm_select (line 97)

Error in spm_jobman (line 173)

Error in spm (line 335)

Error while evaluating DestroyedObject Callback.

I reinstalled both, but it still does not work. When i installed the CONN toolbox with the 2022a matlab runtime it worked without problems. 
Could you help me?

Best regards
Kolja

**Comment by lukeje** [comment]
@Barisevrenugur I don't think that the version of linux is the issue, e.g. it works (excluding some graphical glitches) for me using Ubuntu 22.04 on WSL2. I think it's more likely that something has gone wrong with the installation or selection of the matlab runtime (e.g. maybe the `LD_LIBRARY_PATH` still has the 2022a runtime before the 2023a runtime).

**Comment by SusiStinkemuehle** [comment]
Thank you! It was something wrong with the installation. I unpacked the hMRI zip file within Windows and not Ubuntu. When unpacking within Ubuntu it worked. Best regards, Kolja
